src/main/java/uk/gov/hmcts/darts/event/service/impl/RemoveDuplicateEventsProcessorImpl.java

```diff
@@ -43,7 +49,7 @@ public boolean findAndRemoveDuplicateEvent(Integer eventId) {
         eventRepository.deleteAllById(eventEntitiesIdsToDelete);
         eventRepository.flush();
 
-        log.info("Duplicate events found. Removing the following events {}", eventEntitiesIdsToDelete);
+        log.info("Duplicate events found for eveId {}. Removing the following events {}.", eveIdToKeep, eventEntitiesIdsToDelete);
```

Comment:
Is it possible to log out the message ID of the one affected? Would have made my life so much easier today.